data/remnant outfits.txt

```diff
@@ -359,11 +359,31 @@ outfit "Salvage Scanner"
 
 # Engines:
 
+outfit "Anvil-Class Engine"
+	category "Engines"
+	licenses
+		Remnant
+	"cost" 290000
+	thumbnail "outfit/tiny remnant engine"
+	"mass" 26
+	"outfit space" -26
+	"engine capacity" -26
+	"thrust" 13.5
+	"thrusting energy" 2.65
+	"thrusting heat" 2.5
```

Comment:
The thrust energy and heat shouldn't be exactly the same as the Crucible-Class if the thrust is less. I'd suggest 2.05 energy and 1.95 heat.

Reply:
Alright. Changed